tests/integration/kubernetes/k8s-nginx-connectivity.bats

```diff
@@ -9,6 +9,8 @@ load "${BATS_TEST_DIRNAME}/../../common.bash"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
 
 setup() {
+	[ "${KATA_HYPERVISOR}" == "qemu" ] && skip "test not working see: https://github.com/kata-containers/kata-containers/issues/10414"
```

Comment:
One way to do that would be exposing the `matrix.container_runtime` from https://github.com/kata-containers/kata-containers/blob/main/.github/workflows/run-k8s-tests-on-amd64.yaml#L44 to the `gha-run.sh` script, all the way to the tests ... and skip it if `container_runtime` is `crio`.

WDYT?

Reply:
Yeah, I suspected it :-(
I've tried to look at all other skipped test and didn't find something for crio.

Is it OK if I add an environment variable to expose the `matrix.container_runtime` parameter?
Otherwise I feel my only option is to check the "KUBERNETES_EXTRA_PARAMS", but it could be misleadig/changing in subtle ways in the future.

Reply:
ah, crossed comments, github didn't update fast enough :-p I'll do that